src/DoctrineBatchUtils/BatchProcessing/SimpleBatchIteratorAggregate.php

```diff
@@ -52,7 +52,7 @@ public static function fromArrayResult(array $results, EntityManagerInterface $e
     /**
      * @param Traversable            $results
      * @param EntityManagerInterface $entityManager
-     * @param                        $batchSize
+     * @param int                    $batchSize
```

Comment:
Can we add a type declaration to the method please?

Reply:
done